ui/components/app/connected-accounts-permissions/connected-accounts-permissions.js

```diff
@@ -0,0 +1,83 @@
+import classnames from 'classnames';
```

Comment:
Hey @digiwand : going forward I would suggest avoid refactoring for code cleanup and bug fixing in same PR. This makes it confusing to review.

Reply:
I hear you and I agree. Thanks for the callout! Upon the first round of reviews yesterday, I thought to fix the specific i18n translation, we could simply change 1 line:
```
<label htmlFor={permissionName}>{t(permissionName)}</label>
```

to something like
```
<label htmlFor={permissionName}>
  {permissionName === 'eth_accounts' ? {t('permission_ethereumAccounts')} : t(permissionName)}
</label>
```

However, please correct me if I'm wrong, I believe we realised that this may be a bigger bug. The existing logic would not be able to handle permissions keys that don't map to translations and may break, particularly using flask. We added flask support. To do this, we realised converting the file to a FC and sharing the logic using custom hooks would be cleaner and take roughly the same amount of work as opposed to duplicating code here. @rekmarks would be able to shed more light on this